mysql-test/suite/innodb/t/fk_col_non_null.combinations

```diff
@@ -0,0 +1,5 @@
+[COPY]
+--alter_algorithm=copy
+
+[INPLACE]
+--alter_algorithm=inplace
```

Comment:
I think that there were plans to remove this parameter. It could be better to examine `MTR_COMBINATION_COPY` and `MTR_COMBINATION_INPLACE` in the `.test` file and not specify any parameter here.

A better name for the test would be `foreign_not_null` or `foreign_null`, to better match the already is a SQL phrases `NOT NULL` or `ON…SET NULL`. The `null` already implies that this is about columns.